src/backend/tests/Fusion.Resources.Api.Tests/Fixture/ResourcesApiWebAppFactory.cs

```diff
@@ -103,16 +103,16 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 services.TryRemoveTransientEventHandlers();
                 services.TryRemoveImplementationService("FusionServiceDiscoveryHostedService");
                 services.TryRemoveImplementationService("InfrastructureInitialization");
-                
                 services.TryRemoveImplementationService("PeopleEventReceiver");
                 services.TryRemoveImplementationService("OrgEventReceiver");
                 services.TryRemoveImplementationService("ContextEventReceiver");
+                //services.TryRemoveImplementationService("LineOrgOrgUnitChangedEvent");
                 services.TryRemoveImplementationService<ICompanyResolver>();
 
                 if (IsMemorycacheDisabled)
                 {
                     services.TryRemoveImplementationService<IMemoryCache>();
-                    services.AddScoped<IMemoryCache, AlwaysEmptyCache>();
+                    services.AddTransient<IMemoryCache, AlwaysEmptyCache>();
```

Comment:
we often use memory cache, but I cannot remember to have implemented an AlwaysEmptyCache.
Do you need this ?

Reply:
Other tests chrashes if this is not chagned.  mabye this has to do with the TestDbconnectionstring file having been changed ?

Reply:
These fails if AddScoped are set: 
![image](https://user-images.githubusercontent.com/3900367/211276456-fa4ac7dc-e588-432f-9974-acdfbf108857.png)